2017/DVE-2017-0009.md

```diff
@@ -0,0 +1,150 @@
+# DVE-2017-0009: axc.nl bogus DNSSEC denial of existence
+
+## Description
+
+Some DNSSEC signed zones are served by nszero[12].axc.nl.  The
+problem domains have data only at the zone apex, except possibly a
+child wildcard A record.  In either case the response for a TLSA
+query indicates that only the zone apex exists (even in the presence
+of a wildcard child node).  In both cases the RCODE is NODATA, even
+though the NSEC record "proves" NXDOMAIN (though NODATA would be
+right in the wildcard case, were the NSEC records consistent with
+the presence of the wildcard).
+
+This breaks email delivery from DANE-enabled sending MTAs, since
+bogus TLSA denial of existence is indistinguishable from an active
+downgrade attack.
+
+## Evidence
+
+(RRSIG records and other extraneous dig output elided)
+
+#### NODATA should be NXDOMAIN:
```

Comment:
Level 3 looks much too large IMHO, how important is it to have strictly sequential levels?